.teamcity/builds/kotlinlang/SiteProject.kt

```diff
@@ -0,0 +1,38 @@
+package builds.kotlinlang
+
+import builds.kotlinlang.buidTypes.*
+import builds.kotlinlang.templates.DockerImageBuilder
+import builds.kotlinlang.vcs.GrammarGenerator
+import builds.kotlinlang.vcs.KotlinSpec
+import builds.kotlinlang.vcs.WebHelp
+import jetbrains.buildServer.configs.kotlin.Project
+
+object SiteProject : Project({
+  name = "Build kotlinlang.org"
```

Comment:
Build is neccessery? I still think its better have structure based by project more then function 🙄 like that: 

 - kotlinlang.org
    - build
    - deploy

But it's unnecessary, we can discuss about somewhere else 🙄

Reply:
Yeah, I agree with your proposal. 
I've fixed the name, but the structure itself in DSL right now is a bit different. I suggest to return to aligning the structure when the first deploy is created.